### DISCUSSION TITLE
Hi everyone

### DISCUSSION THREAD

**Discussion by MarcePSS** [Green-Software-Foundation/discussions#59] [Introductions]
Hi!👋 I'm Marcela (She/Her). I'm delivery engineer at Accenture.
I have only recently become acquainted with green software. Almost by chance I found out that it was possible to contribute to sustainability through software and although I am not a developer I immediately started to try to learn about it.
Now I want to measure the SCI of the developments I can access in my company and present them here as case studies.
You can contact me on LinkedIn www.linkedin.com/in/marcelapazsepulveda

**Comment by NAMRATA-WOKE** [comment]
Welcome @MarcePSS! Will you be joining us at [Decarb 2023](https://grnsft.org/decarb)?

**Reply by MarcePSS** [reply]
Yep! thanks!, I'm signing up now.
